### ISSUE TITLE
Git clone is using the wrong URL

### ISSUE THREAD

**Issue by mdeeks** [MatthewJohn/terrareg#5]
Trying to create a module using the required `ssh://git@github.com:org/terraform-s3-bucket.git` format but it consistently fails to index in the UI. It gives this error message:
```Error occurred during git clone: fatal: Could not read from remote repository.```

I went in and added a print to GitModuleExtractor._clone_repository() and the ssh command looks like this:
```
terrareg_1  | ['git', 'clone', '--single-branch', '--branch', 'v6.0.5', 'ssh://git@github.com:org/terraform-s3-bucket.git', '/tmp/tmpo1yx9kao']
```
The `ssh://` prefix is messing up the command. If I exec into the container and run the command manually I see this error which matches

```
git clone --single-branch --branch v6.0.5 ssh://git@github.com:org/terraform-s3-bucket.git
Cloning into 'terraform-s3-bucket'...
ssh: Could not resolve hostname github.com:org: Name or service not known
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
I confirmed I am mounting my ssh key and I can manually clone in the container

**Comment by MatthewJohn** [comment]
Hey @mdeeks ,

Thank you raising this.

From testing and seeing what we've used in the past for the URL format for SSH is that the URL format needs to be: `ssh://github.com/org/name.git`. Since we enforce the 'ssh://' protocol on the URL, the colon before the organisation won't be used and is treated as a port.
The examples/placeholders appear to all be incorrect, so I will fix them and will investigate adding support for the format that Github provides (`git@github.com:org/name`) in the clone instructions.
I'll add some checks to ensure that the URL matches a working format.

Many thanks
Matt

**Comment by MatthewJohn** [comment]
Hey @mdeeks ,

I've raised a PR to:
 * Update the placeholder in the page
 * Add checks to ensure the URL is not passed in this format

https://gitlab.dockstudios.co.uk/pub/terrareg/-/merge_requests/315

I'll hopefully get this merged in the next hour, but if you want to test in the meantime, hopefully changing the format of the URL should fix your problem :)

Thanks
Matt

**Comment by MatthewJohn** [comment]
I've also created https://gitlab.dockstudios.co.uk/pub/terrareg/-/issues/362 to investigate supporting the URL format that Github provides

Many thanks
Matt

**Comment by mdeeks** [comment]
Thanks! that looks like it worked. I thought it was the `ssh://` prefix but that is in fact a valid clone url. I'm guessing that github CLI has a special case for when you try to clone with the github format of `git@github.com:/org/repo.git` syntax? Anyway, if I change our GIT_PROVIDER_CONFIG I can make this easy for our users.

I am able to successfully index multiple projects from github now.

**Comment by MatthewJohn** [comment]
> Thanks! that looks like it worked.
That's good :) 

> I'm guessing that github CLI has a special case for when you try to clone
I'm not entirely sure, but as far as I know, Git can use the same ssh-style syntax (same as rsync), but initially, since we need to dissect parts of the path, only supported URL-style.

> Anyway, if I change our GIT_PROVIDER_CONFIG I can make this easy for our users.
> I am able to successfully index multiple projects from github now.

Excellent :)

**Comment by MatthewJohn** [comment]
Changes for this were released in v2.62.2
